test/features/case.ts

```diff
@@ -20,12 +20,12 @@ export default function supportsCase(format: FormatFn) {
 
   it('formats CASE ... WHEN with an expression', () => {
     const result = format(
-      "CASE toString(getNumber()) WHEN 'one' THEN 1 WHEN 'two' THEN 2 WHEN 'three' THEN 3 ELSE 4 END;"
+      "CASE trim(sqrt(2)) WHEN 'one' THEN 1 WHEN 'two' THEN 2 WHEN 'three' THEN 3 ELSE 4 END;"
```

Comment:
Here and in several other tests I've switched over to using the some widely adopted function names, so the tests would work across all dialects.